deploy/runtools/runtime_config.py

```diff
@@ -244,8 +244,18 @@ def __init__(self, runtimeconfigfile, configoverridedata):
             rootLogger.warning(overridefield + "=" + overridevalue)
             runtime_dict[overridesection][overridefield] = overridevalue
 
-        runfarmtagprefix = "" if 'FIRESIM_RUNFARM_PREFIX' not in os.environ else os.environ['FIRESIM_RUNFARM_PREFIX'] + "-"
+        runfarmtagprefix = "" if 'FIRESIM_RUNFARM_PREFIX' not in os.environ else os.environ['FIRESIM_RUNFARM_PREFIX']
```

Comment:
Remind me, how is this being set in the environment?

Reply:
This is set by the user if they want, we don't set it anywhere explicitly. It's just to help users keep track if they have multiple managers/clones of firesim.

Reply:
(this feature was added before, I just fixed a bug where if you don't have this env var set, your runfarmtag would start with a "-")